pkg/system/filesys_windows.go

```diff
@@ -0,0 +1,64 @@
+// +build windows
+
+package system
+
+import (
+	"os"
+	"regexp"
+	"syscall"
+)
+
+// MkdirAll implementation that is volume path aware for Windows.
+func MkdirAll(path string, perm os.FileMode) error {
+	if re := regexp.MustCompile(`^\\\\\?\\Volume{[a-z0-9-]+}$`); re.MatchString(path) {
+		return nil
+	}
+
+	// The rest of this method is copied from os.MkdirAll and should be kept
+	// as-is to ensure compatibility.
```

Comment:
@jhowardmsft isn't this MkdirAll function just a wrapper around os.MkdirAll ? If that's the case why not just call os.MkdirAll instead of copying the code (which btw differs from latest stable: https://golang.org/src/os/path.go#L19)